src/components/Pathways.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-
+import { Link } from "react-router-dom";
 
 function Pathways() {
   return (
     <div class="pathways">
-      <a href="#Artists">Artists</a>
+      <Link to='/artist'>Artists</Link>
```

Comment:
Siguiendo mi comentario de plurales:
```suggestion
      <Link to='/artists'>Artists</Link>
```